src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -2404,6 +2404,8 @@ void mvnw(Register Rd, Register Rm,
   INSN(umlalv, 1, 0b100000, false); // accepted arrangements: T8B, T16B, T4H, T8H, T2S, T4S
   INSN(maxv,   0, 0b011001, false); // accepted arrangements: T8B, T16B, T4H, T8H, T2S, T4S
   INSN(minv,   0, 0b011011, false); // accepted arrangements: T8B, T16B, T4H, T8H, T2S, T4S
+  INSN(smaxp,  0, 0b101001, false); // accepted arrangements: T8B, T16B, T4H, T8H, T2S, T4S
+  INSN(sminp,  0, 0b101011, false); // accepted arrangements: T8B, T16B, T4H, T8H, T2S, T4S
```

Comment:
Just one nit: do we need an assembler test case for them?

Reply:
Sure, added in the updated commit. Thanks.